soccer/planning/IndependentMultiRobotPathPlanner.cpp

```diff
@@ -33,6 +35,14 @@ std::map<int, std::unique_ptr<Path>> IndependentMultiRobotPathPlanner::run(
         staticRobotObstacles[shell] = std::make_shared<Geometry2d::Circle>(
             request.start.pos, Robot_Radius);
     }
+
+    auto comparator = [&](int &lhs, int &rhs) {
+            return sortByPriority(requests[lhs], requests[rhs]);
+        };
+
+    std::sort(std::begin(staticRequests), std::end(staticRequests), comparator);
+    std::sort(std::begin(dynamicRequests), std::end(dynamicRequests), comparator);
+
     std::vector<int> inOrderRequests;
```

Comment:
Does it make sense to always plan the static requests first?  Won't this allow a low-priority robot with a static request to be planned before a high-priority robot with a dynamic request?